src/Components/ServiceScene/Breakdowns/PatternsFrameScene.tsx

```diff
@@ -20,14 +20,16 @@ import { IndexScene } from '../../IndexScene/IndexScene';
 import { PatternsViewTableScene } from './PatternsViewTableScene';
 import { config } from '@grafana/runtime';
 import { css } from '@emotion/css';
+import { PatternsBreakdownScene } from './PatternsBreakdownScene';
 
 const palette = config.theme2.visualization.palette;
 
 export interface PatternsFrameSceneState extends SceneObjectState {
   body?: SceneObject;
   loading?: boolean;
-  patternFrames: PatternFrame[];
+  patternFrames?: PatternFrame[];
```

Comment:
I'll take another look, if I remember correctly this was done because sometimes the patternsFrameScene is instantiated in the breakdown scene in a subscription, and in that case the parent doesn't have the new frame set to state.

Reply:
Ok, so we still need to pass in the patternFrames into the PatternsViewTableScene because the parent scene decides if the children get the filtered copy of the patterns or not. Otherwise we need to do even more state checking from various classes. But I did remove it from this class and now it's just the breakdown scene and the children. I'll add some comments